http_service/bugbug_http/models.py

```diff
@@ -159,29 +159,47 @@ def schedule_tests(branch, rev):
     # Autoland should always rebase on top of parents, never on tip
     default_base = "tip" if branch != "integration/autoland" else None
     try:
-        apply_stack(REPO_DIR, stack, branch, default_base)
+        first_rev = repository.apply_stack(REPO_DIR, stack, branch, default_base)
     except Exception as e:
         LOGGER.warning(f"Failed to apply stack {branch} @ {rev}: {e}")
         return "NOK"
 
-    first_rev = stack[0]["node"]
-    if first_rev != rev:
-        revset = f"{first_rev}::{rev}"
```

Comment:
I still think it would be nice to support revsets here so we can more easily use the commit classifier in other contexts, but it's definitely something we can implement later on when/if we need it.